dashboard/src/controllers/widgets/channel-list-widget-controller.js

```diff
@@ -50,15 +50,19 @@ window.Discotron.ChannelListWidgetController = class extends window.Discotron.Wi
      * Add a list of channel that can be selected to the modal
      */
     _displayChannels() {
-        for (let i = 0; i < this._channels.length; i++) {
-            const channel = this._channels[i];
+        for (let discordId in this._channels) {
+            const channel = this._channels[discordId];
+            if (channel.type !== "text") {
```

Comment:
I would rather exclude *only* category channels, instead of *only* accepting text channels. Bot could also in theory respond in store/news channels, as well as group and DM chats.